charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -16,3 +16,9 @@ maintainers:
     email: chengpan@amazon.com
   - name: krmichel
     url: https://github.com/krmichel
+annotations:
+  artifacthub.io/changes: |
+    - kind: changed
+      description: Use chart app version as default image tag
+    - kind: changed
+      description: Add updateStrategy to daemonsets
```

Comment:
i'm curious what purpose these annotations serve when we already have a helm chart changelog

Reply:
They will show up on artifact.io when the issue to publish there gets completed